### PR TITLE
Revised `Discussion`.

### PR DESCRIPTION
**Pull request by m1cm1c**
I wasn't able to wait until everyone did their changes. Otherwise we won't finish today. I know that this has conflicts but resolving them will be faster than having waited and started later.

### CODE REVIEW

**Comment by m1cm1c** [comment]
Resolved conflicts.

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Discussion.lyx [hunk 1]
```diff
@@ -191,11 +191,11 @@ name "/C20/"
...
- The measured software has a constant, deterministic runtime for a fixed
- configuration of input parameters, when ignoring influences of the hardware,
- operating system and error of measurement.
- This will be the case for most software.
- The fact a user tries to measure the software when using 
+ The measured software system has a constant, deterministic runtime for
+ a fixed configuration of input parameters when ignoring influences of the
+ hardware, the operating system, and the error of measurement.
+ This will be the case for most software systems.
+ The fact that the users try to measure the software system when using
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Discussion.lyx]: users ?!

File: doc/Requirements Specification/Parts/Discussion.lyx [hunk 2]
```diff
@@ -234,7 +234,7 @@ reference "input artefacts"
 
 \end_inset
 
-) are integer.
+) are integers.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Discussion.lyx]: No! The input artefacts are not integers.

File: doc/Requirements Specification/Parts/Discussion.lyx [hunk 3]
```diff
@@ -248,7 +248,7 @@ gls{PCM}
 
 \end_inset
 
- describe the software correctly, completely and exactly like implemented
+ describe the software correctly, completely, and exactly like implemented
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Discussion.lyx]: Are you sure, that there has to be a comma?
In german, no comma would be needed.

File: doc/Requirements Specification/Parts/Discussion.lyx (hunk 2 above)
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Discussion.lyx]: :D

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Discussion.lyx]: [“of integrity” seems to be the correct translation](http://dict.leo.org/ende/index_de.html#/search=integer&searchLoc=0&resultOrder=basic&multiwordShowSingle=on)

Sorry for the confusion! Didn’t laugh so hard for a long time.

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Discussion.lyx]: but maybe something like “were not modified” would fit best here.

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Discussion.lyx]: I was confused by this for a week now.

File: doc/Requirements Specification/Parts/Discussion.lyx (hunk 1 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Discussion.lyx]: Fixed it.

File: doc/Requirements Specification/Parts/Discussion.lyx (hunk 3 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Discussion.lyx]: That's the Oxford Comma.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**